packages/feedback/src/app/app.component.spec.ts

```diff
@@ -0,0 +1,28 @@
+import { TestBed, async } from '@angular/core/testing';
+import { RouterTestingModule } from '@angular/router/testing';
+import { AppComponent } from './app.component';
+
+describe('AppComponent', () => {
+  beforeEach(async(() => {
```

Comment:
No need for async here. `async` is useless without an await.

```suggestion
  beforeEach(() => {
```

Reply:
removed.